test/controllers/observations/identify_controller_test.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+module Observations
+  class IdentifyControllerTest < FunctionalTestCase
+    def test_identify_observations_index
```

Comment:
I personally like (and I think @mo-nathan likes) one request per controller test method.
This reads more like an integration test.

Reply:
True. I wanted to write an integration test, but the JS blocks that for now. 

This seemed DRYer, although i know there's a way to refactor so that the repetitive steps are performed each time. That has some advantages, but it also makes the test slower. I could be convinced, though.

Reply:
(The issue, by the way, is that I have to make two requests for each test... because what i'm testing is if underlying alterations to the db alter the query results. So I need a before/after. Maybe i should move most of this to an integration test? But it's not strictly an integration test since it does the db alterations directly.)